### ISSUE TITLE
Necessary Hardware for SteamVR games like Half-Life: Alyx

### ISSUE THREAD

**Issue by deniz36** [psmoveservice/PSMoveService#687]
Sorry for the inconvenience, I try to get into VR for long time now, but the topic seems so obscure.
Now I found your git and I have one PS Move controller laying around from past days, so if I want to go full VR with PSMoveService, from my research, what I need will be:

- 2 PS Move Motion Controllers
- 2 PS Move Navi Controllers
- 2 Joiner Clips
- 1 Kinect 360
- 1 Gear VR Headset (or similar)
- 1 extra Bluetooth adapter
- Diver4VR

Now, without the Move Controller I already own, this will leave me at about €190EUR. With the prices of conventional VR Kits being absurdly high at the moment, I wonder wether it is worth going for the PS Move setup or waiting another few months would be the better option.

Thanks in advance.

**Comment by el-bohno** [comment]
You can use ps3 eye cams, maybe 4 are cheaper than 1 kinect 360, and the tracking is much better. But a little bit more setup to do. I´m doing it with intern onboard Bluetooth adapter. At the moment there is no way to play half life alyx with psmove, attracting things seems to dont work.  Don´t know if navi controllers work with Driver4vr, but with 3 ps move controllers you are ready to go. ( 1+ for head tracking). If you would like to play with kinect 360 you need only 2 ps move controller.  To move with  sticks, you can use that, too https://de.aliexpress.com/item/32857382935.html?src=google&src=google&albch=shopping&acnt=494-037-6276&isdl=y&slnk=&plac=&mtctp=&albbt=Google_7_shopping&aff_platform=google&aff_short_key=UneMJZVf&&albagn=888888&albcp=9317151805&albag=93621837065&trgt=299423776478&crea=de32857382935&netw=u&device=c&albpg=299423776478&albpd=de32857382935&gclid=EAIaIQobChMI49Gr5Ky76AIVBLDtCh2Hcw7QEAQYASABEgItAvD_BwE&gclsrc=aw.ds

**Comment by MarkJGx** [comment]
It's not worth. Unfortunately the tracking quality is really lackluster even with the ideal setup. I've tried PSMoveService several times over the past 4 years and sure the tracking has improved, but it's not a playable/usable level. I have tried out several sized playspaces, from medium to large, hours of finicky setup. Result was in no way playable. Tracking inaccurate and drifty.

**Comment by AgoniXX** [comment]
Right, half life alyx is not working here either. The key binding doesn't work at all.

**Comment by Di1Ly** [comment]
Just start HLVR with PSVR+P3Eye+2xPSMove.
I can see hands and I can point laser to menu but trigger is not working and other buttons to.
When I try to open Binding I have only "Loading current binding..."
Also I have an error in HLVR debug console - "Unknown controller type name 'playstation_move' - falling back to HMD manufacturer name."

Can I do something to play HLVR?)))

**Comment by AgoniXX** [comment]
Same here.
No way yet.
Just download the non-vr mod, and play with PSVR and mouse and keyboard.

**Comment by Di1Ly** [comment]
PSVR (iVRy) + Kinect (Driver4VR) + 2xPSMove(PSMoveService) + HL Alyx = more or less playable.

**Comment by AgoniXX** [comment]
> PSVR (iVRy) + Kinect (Driver4VR) + 2xPSMove(PSMoveService) + HL Alyx = more or less playable.

So the binding works with Driver4VR? I'll try it.

**Comment by Zeronucks** [comment]
I've been able to get through a large section of the game just using PSmoveService and Vridge, I've had to manually set all the binding's and the game has been playing perfect aside from the Throwing and Gravity Gloves, I've managed to bypass the tutorial that requires you to use the Russell's by giving myself a gun through Dev Cheats, I've been trying multiple ways of getting the Russell's to work but it seems until this program gets updated we won't have any way of fully using the Gravity Gun and seeing how the last update was in 2018 I don't believe we're gonna get any more help, unless we throw some cash at a Kinect and Driver4VR.

**Comment by DimMario** [comment]
use 1.5.2 steamvrbridge this fix for trigger not work

**Comment by AgoniXX** [comment]
> use 1.5.2 steamvrbridge this fix for trigger not work

Where to dl 1.5.2? Only has 1.5.1 or 1.6
https://github.com/HipsterSloth/PSMoveSteamVRBridge/releases
